packages/contracts/contracts/L1/rollup/OVM_CanonicalTransactionChain.sol

```diff
@@ -37,6 +37,8 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
     uint256 constant public MIN_ROLLUP_TX_GAS = 100000;
     uint256 constant public MAX_ROLLUP_TX_SIZE = 50000;
     uint256 constant public L2_GAS_DISCOUNT_DIVISOR = 32;
+    uint256 constant public ENQUEUE_GAS_CONSUMED = 55000;
```

Comment:
Justification for this value based on the [measurement here](https://dashboard.tenderly.co/tx/mainnet/0xdf2ca1773f9682df29748ad519cebba7aadde79cd7350741180f60816766ccd4/gas-usage):
* The call to `enqueue` consumes about 116k gas
* About 46.8% of that is 'other' operations not accounted for by the CALL operations:
![image](https://user-images.githubusercontent.com/23033765/133503208-16ac9da1-bfbe-4eff-a6f7-3be54254b86c.png)
* Gas used in `enqueue` without the burn: `116*(1-0.47) =~ 62000`.
* Round down to 60k for some safety margin.